lib/bundler/cli.rb

```diff
@@ -465,10 +465,14 @@ def platform
     end
 
     desc "inject GEM VERSION", "Add the named gem, with version requirements, to the resolved Gemfile"
+    method_option "source", :type => :string, :banner =>
```

Comment:
Maybe `"install gem from the given source"` and `"Install gem into a bundler group"`